plugins/inputs/snmp/snmp_mocks_test.go

```diff
@@ -0,0 +1,82 @@
+package snmp
```

Comment:
I split all the command mocking stuff out of `test_snmp.go` to keep the tests clean.